util/parseOptions.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Parse options string into object.
+ *
+ * @param {string} optionsString
+ * @returns {{}}
+ */
+module.exports = (optionsString) => {
+  let options = {};
```

Comment:
This can be a `const`. `const` only protects against re-assignment, but the properties of the object may still be modified.